extensions/amp-iframely/OWNERS.yaml

```diff
@@ -1,4 +1,5 @@
 # For an explanation of the OWNERS.yaml rules and syntax, see:
 # https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example.yaml
 
+- OmriKeret
```

Comment:
This addition appears to modify `amp-iframely` which is otherwise not relevant for this PR. Can this change be made in a separate PR please?

Reply:
Yes, we'll place this change in the new PR.